src/components/Image/index.scss

```diff
@@ -2,6 +2,27 @@
 
 .Image {
 
+  &-container {
+    height: 100%;
+    position: relative;
+    width: 100%;
```

Comment:
You don't need this property, the default width value for a `<div>` element is 100% of the container.